biodatasets/n2c2_2018_track1/n2c2_2018_track1.py

```diff
@@ -0,0 +1,296 @@
+    # ds_bigbio = datasets.load_dataset('my_dataset', name='bigbio', data_dir="/path/to/data/files")
+
+    _SOURCE_CONFIG_NAME = _DATASETNAME + "_" + SOURCE
+    _BIGBIO_CONFIG_NAME = _DATASETNAME + "_" + BIGBIO_TEXT
+
+    BUILDER_CONFIGS = [
+        BigBioConfig(
+            name=_SOURCE_CONFIG_NAME,
+            version=SOURCE_VERSION,
+            description=_DATASETNAME + " source schema",
+            schema=SOURCE,
+            subset_id=_DATASETNAME,
+        ),
+        BigBioConfig(
+            name=_BIGBIO_CONFIG_NAME,
+            version=BIGBIO_VERSION,
+            description=_DATASETNAME + " BigBio schema",
+            schema=BIGBIO_TEXT,
+            subset_id=_DATASETNAME,
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = _SOURCE_CONFIG_NAME
+    LABEL_CLASS_NAMES = [
+        "ABDOMINAL",
+        "ADVANCED-CAD",
+        "ALCOHOL-ABUSE",
+        "ASP-FOR-MI",
+        "CREATININE",
+        "DIETSUPP-2MOS",
+        "DRUG-ABUSE",
+        "ENGLISH",
+        "HBA1C",
+        "KETO-1YR",
+        "MAJOR-DIABETES",
+        "MAKES-DECISIONS",
+        "MI-6MOS",
+    ]
+
+    def _info(self) -> datasets.DatasetInfo:
+
+        if self.config.schema == SOURCE:
+            labels = {
+                key: datasets.ClassLabel(names=["met", "not met"])
+                for key in N2C22018CohortSelectionDataset.LABEL_CLASS_NAMES
+            }
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "document_id": datasets.Value("string"),
+                    "text": datasets.Value("string"),
+                    "tags": labels,
+                }
+            )
+
+        elif self.config.schema == BIGBIO_TEXT:
+            features = schemas.text.features
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager) -> List[datasets.SplitGenerator]:
+
+        if self.config.data_dir is None or self.config.name is None:
+            raise ValueError("This is a local dataset. Please pass the data_dir and name kwarg to load_dataset.")
+        else:
+            data_dir = self.config.data_dir
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                # Whatever you put in gen_kwargs will be passed to _generate_examples
+                gen_kwargs={
+                    "file_path": os.path.join(data_dir, "train.zip"),
+                    "split": str(datasets.Split.TRAIN),
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.TEST,
+                gen_kwargs={
+                    "file_path": os.path.join(data_dir, "n2c2-t1_gold_standard_test_data.zip"),
+                    "split": str(datasets.Split.TEST),
```

Comment:
these kwargs need to be consistent with what `_generate_examples` accepts.

Reply:
done